src/libs/actions/Report.js

```diff
@@ -204,13 +204,14 @@ function fetchChatReportsByIDs(chatList) {
  *
  * @param {Number} reportID
  * @param {Number} sequenceNumber
+ * @param {Object} report
  */
-function setLocalLastRead(reportID, sequenceNumber) {
+function setLocalLastRead(reportID, sequenceNumber, report = null) {
```

Comment:
Maybe it would be simpler to just pass the `maxSequenceNumber` rather than the whole report object?